Formula/moc.rb

```diff
@@ -2,6 +2,7 @@ class Moc < Formula
   desc "Terminal-based music player"
   homepage "https://moc.daper.net/"
   revision 6
+  license "GPL-2.0-or-later"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
  revision 6
```

Reply:
I had the two lines in the correct order in the file locally so `brew audit --strict moc` returned no issues for me, but I had committed the two lines in the wrong order. :/